lib/sync/datasetClientsCleaner.js

```diff
@@ -22,26 +23,42 @@ var storage;
 function DatasetClientsCleaner(opts) {
   this.retentionPeriod = opts.retentionPeriod || '24h';
   this.checkFrequency = opts.checkFrequency || '1h';
+  this.cleanerLockName = opts.cleanerLockName || 'locks:sync:DatasetCleaner';
```

Comment:
Is `opts.cleanerLockName` able to be set by the user? It looks like it's not an option in sync-server and it not passed in?